man/systemd.network.xml

```diff
@@ -1004,6 +1004,24 @@ DuplicateAddressDetection=none</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MulticastIGMPVersion=</varname></term>
+        <listitem>
+          <para>Configure IPv4 Multicast IGMP Version. If not configure not change the IPV4 Multicast IGMP version.
+          If configured it takes one of
+          <literal>no</literal>, <literal>1</literal>, <literal>2</literal>, or <literal>3</literal>.
+          When <literal>no</literal> no enforcement of a IGMP version, IGMPv1/v2 fallback allowed, will back to
+          IGMPv3 mode again if all IGMPv1/v2 Querier Present timer expires no source validation will be done.
+          When <literal>1</literal> enforce to use IGMP version 1, will also reply IGMPv1 report if receive
+          IGMPv2/v3 query.
+          When <literal>1</literal> enforce to use IGMP version 2, will fallback to IGMPv1 if receive IGMPv1 query
+          message. Will reply report if receive IGMPv3 query.
+          </para>
+
```

Comment:
i think i'd be more comfortable if these enums would be called "v1", "v2", "v3", if they indicate versions.

Simply because versions are a high-level concept, and numer enum indexes a low-level. Here they coincide, but i don't think we should encode that in the naming really.

Reply:
Ok. I update the pull request